internal/storage/download_test.go

```diff
@@ -56,8 +67,59 @@ func TestDownloadToFile(t *testing.T) {
 			HashType:  "SHA256",
 			HashValue: "4eefb9a7a40a8b314b586a00f307157043c0bbe4f59fa39cba88773680758bc3",
 		},
+	}, "", "", t)
+}
+
+// TestDownloadToFileSecureSystemPool tests downloadToFile function, using secure protocol(s) and certificates from system pool.
+func TestDownloadToFileSecureSystemPool(t *testing.T) {
+	// SystemCertPool does not run on Windows:
+	//https://github.com/golang/go/issues/16736
```

Comment:
In general, the comments should use one and the same formatting i.e. there should be a space.